how-to/offline_cpu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Description: Offline SOS CPUs except BSP before launch UOS
+
+for i in `ls -d /sys/devices/system/cpu/cpu[1-9]*`; do
+        online=`cat $i/online`
+        idx=`echo $i | tr -cd "[0-9]"`
+        echo cpu$idx online=$online
```

Comment:
Nits:

- I'd double quote the string.
- Maybe add an "INFO: $0: " prefix?

Reply:
Can you clarify why this is needed?

Reply:
The prefix isn't *needed* but is a sort of loose standard we have adopted whereby we prefix the message level to log output in most areas.

The quoting is simply best practice for defensive shell programming to minimise the chance of unexpected side effects.